valet

```diff
@@ -52,9 +52,17 @@ then
         fi
     done
 
+    # Determine which region to pass along to ngrok...
```

Comment:
This can just be `REGION=${2:-"us"}`, which will take the second argument as the region, using `us` as the default value.